src/features/landing/components/Footer/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Stack, Box, Container } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+import Logo from '../../assets/svg/Logo/Logo.svg';
+
+import CopyrightNoticeAndLinks from './CopyrightNoticeAndLinks/CopyrightNoticeAndLinks';
+import PrivacyPolicy from './PrivacyPolicy/PrivacyPolicy';
+
+function Footer() {
+  return (
+    <Stack
+      sx={{
+        borderTop: '1px solid  #E1E7EA)',
+        background: '#FFF',
+        boxShadow: '0px -5px 46px 0px rgba(198, 209, 220, 0.25)',
+      }}
+    >
+      <Box sx={{ paddingTop: '11px', paddingBottom: '6px' }}>
+        <Container
```

Comment:
why here Container component? why you didn use Grid component with property container?